src/main/kotlin/entity/ChattORESpec.kt

```diff
@@ -46,5 +48,8 @@ object ChattORESpec : ConfigSpec("") {
         val leaveDiscord by optional("**<player> has left the network**")
     }
 
-    val nicknamePresets by optional(pridePresets)
+    val nicknamePresetsPride by optional(pridePresets)
```

Comment:
My thought on the map of maps idea for organizing presets could make the resulting YAML turn into something like this:

```yaml
nickname_presets:
    pride:
        one: 'preset'
        two: 'preset'
    birdflop:
        ....
    players:
        ....
    anotherType:
        one: 'preset'
        two: 'preset'
````

Reply:
What abt turning all 3 example preset files into 1 big presets.kt file with this structure